### DISCUSSION TITLE
child selectors in CSS

### DISCUSSION THREAD

**Discussion by DuajDiaz** [DuajDiaz/Travel-Website#55] [Q&A] [answered]
What are the uses of child selectors in CSS?

**Comment by Bunny77K9** [comment, marked as answer]
Child selectors are primarily used in CSS to look up the ‘child’ component of an element in CSS.

Consider an example where the <ul> tag is used in a paragraph. Then, the ‘ul’ tag becomes a child of the paragraph element. To implement this in CSS, the following syntax is used:

p > ul { font-size:20px; }
